modules/040-node-manager/crds/staticinstance.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       name: v1alpha1
       schema:
         openAPIV3Schema:
-          description: StaticInstance is the Schema for the staticinstances API
+          description: StaticInstance is the Schema for the Cluster API Provider Static
```

Comment:
i think this is not a schema for CAPS. This is a way to describe static machines in the cluster

Reply:
Updated